runtime-infra/pn-cache.yaml

```diff
@@ -186,10 +190,6 @@ Outputs:
   AllowedSecurityGroupId:
     Description: The id of the security group that can access the cache cluster
     Value: !GetAtt AllowedSecurityGroup.GroupId
-
-  RedisEndpoint:
```

Comment:
@mbrachi questo output è necessario per il corretto deploy di pn-logextractor-be